wg-meetings/2020-03-25.md

```diff
@@ -0,0 +1,93 @@
+ * Gireesh: wondering people who are not in the call what do they think, like Michael
+ * Gireesh: will ask Michael
+ * Action item: ping people on an issue (Gireesh)
+
+## Announcements
+ 
+*Extracted from **diag-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/diagnostics
+
+* reportVersion semantics are not defined [#349](https://github.com/nodejs/diagnostics/issues/349)
+  * Matheus going to propose schema based “versioning”
+  * Report was promoted to experimental, does it change this conversation?
+  * Gireesh I don’t think it changes conversation
+  * Mattheus will comment on issue
+* Proposal to drive Diagnostics WG initiatives through user journeys [#295](https://github.com/nodejs/diagnostics/issues/295)
+  * Doing CPU profiling on next WG meeting
+  * How to popularize user journeys
+  * There was an idea to make it part of the official Node.js documentation as it is the main place where people look for help. Who would be the right person to make such a decision?
+* There is a website repository, probably opening an issue there
+* Other idea was to run it on upcoming conferences
+* Is OpenJS still happening?
+* We are waiting for news on that
+* What would be the way to popularize it at conferences?
+* Gireesh: we could propose session for the collaborator summit
+* Maybe we could turn user-journeys and symptom discovery to an interactive website that guides you through
+* Matheus: we discovered a gap in user-journey around debugging a native addon. When I write an addon there is no good tool to debug it mainly because currently recommended tools would require a debug build of Node. Probably llnode is the closest. I think it’s a gap in both user journeys and in tools.
+* Peter: how should we handle it? Discuss next meeting?
+* Mattheus: yes discussing on next meeting + opening a PR to add to empty files
+* Action items: reach out to website repo [Peter], collaborator summit submissions (will write a draft and discuss with WG) [Gireesh], Opening a PR about native addon use-case and gap [Matheus]
+
+* Diagnostics "Best Practices" Guide? [#211](https://github.com/nodejs/diagnostics/issues/211)
+* As part of user journeys we also provide best practices we feel we can collapse the two and remove this from the agenda
+* Worth bringing it back if user journey outcomes doesn’t make it to the docs (PR’s are not open or merged)
+
+* \[async_hooks\] stable API - tracking issue [#124](https://github.com/nodejs/diagnostics/issues/124)
+* A benchmark was added by @legendecas
+* Promises are slow, currently trying to tackle it
+* Is Promises the last missing piece?
+* Not sure, but maybe API still needs some work
+* Gireesh: async local storage API I think will require refining
+* Gireesh: async hooks and async local storage would go hand in hand? Like would they come out together from experimental? I would expect async local storage is based on async-hooks
```

Comment:
The purpose of creating the Async Local Storage API was to provide a higher level API that we hoped could become non-experimental sooner than Async Hooks as it exposes less internals.  So tying them together I don't think makes sense.

Reply:
acknowledged. I guess there is nothing actionable here in the PR (as the line in question is part of a conversation, and does not need to be amended?)